nuxt.config.ts

```diff
@@ -31,6 +32,9 @@ export default defineNuxtConfig({
       systemName: process.env.ELECTRON ? "Electron" : "Web",
     },
   },
+  plausible: {
```

Comment:
@BeastiW - what's the reason for configuring the domain here? On the documentation (https://github.com/nuxt-modules/plausible#basic-usage) I read:
> By default, @nuxtjs/plausible will use window.location.hostname for the Plausible domain configuration key, which will suit most users.

What was the reason for specifying the domain?

Reply:
The reason: I did not think about the reason ;-) I guess window.location.hostname should be enough. I just took it from the issue description. I can remove it and create another PR